src/components/ConfettiOverlay/ConfettiOverlay.js

```diff
@@ -0,0 +1,29 @@
+const defaultCountOfParticles = 80;
+const maximumCountOfParticlesForSmoothPerformance = 200;
+
+const props = {
+  particles: {
+    type: [Number, String],
+    default: defaultCountOfParticles,
+    validator: (value) => {
+      const numericValue = Number.parseInt(value, 10);
+      const isValid = !Number.isNaN(numericValue);
+      return (isValid && numericValue <= maximumCountOfParticlesForSmoothPerformance);
```

Comment:
(nit) please update this with a simple return, and enclose the second clause within parens, like so: 

```js
const result = (isValid && (numericValue <= maximumCountOfParticlesForSmoothPerformance));
return result;
```